kubernetes/polkassembly/values-dashboards-cluster-1.yaml

```diff
@@ -13,12 +13,17 @@ frontend:
   ingress:
```

Comment:
we should actually remove the dependency to the dashboards cluster, but that's not the scopen of the issue here.